src/Uno.UI/UI/Xaml/Controls/WebView/Native/iOSmacOS/UnoWKWebView.iOSmacOS.cs

```diff
@@ -267,7 +267,7 @@ private void OpenUrl(string url)
 		var nsUrl = new NSUrl(url);
 		//Opens the specified URL, launching the app that's registered to handle the scheme.
 #if __IOS__
-		UIApplication.SharedApplication.OpenUrl(nsUrl);
+		Task.Run(() => UIApplication.SharedApplication.OpenUrlAsync(nsUrl, new UIApplicationOpenUrlOptions()));
```

Comment:
What's the min version for this specific API? We may need to gate this behind a version check.

Reply:
@jeromelaban @MartinZikmund From looking at the Apple docs, I think it should be (iOS 10.0+, iPadOS 10.0+), so I think it should not be a problem, since we support iOS 11+. I couldn't test this PR yet, as I don't have an Apple device, but Martin promised to do so, to be sure.